### ISSUE TITLE
[hint-compat-api] Ignore unsupported CSS values when basic support is missing

### ISSUE THREAD

**Issue by antross** [webhintio/hint#1589]
(Found testing the compat-api hint on webhint.io using the modified browser extension from #1574).

Log:
```
hint 10: appearance prefixed with -webkit- is not supported on firefox 60, ie browsers.
https://webhint.io/static/styles/common-79e0384215.css:1:24966

hint 11: none is not supported on ie browser.
https://webhint.io/static/styles/common-79e0384215.css:1:24966
```

Source:
```css
    -webkit-appearance: none;
    -moz-appearance: none;
    appearance: none;
```

Here since "ie" is already reported as not supporting "appearance", we should not be separately logging the missing support for the "none" value. 

Also worth taking a look as to why "firefox 60" is getting flagged here since the MDN data shows it as having support since version 1.

**Comment by borgitas21** [comment]
The fix should included as part of #1584

**Comment by borgitas21** [comment]
@antross this issue should be fixed as part of #1584 . Could you check it?

**Comment by antross** [comment]
@borgitas21 Yes, this was fixed by #1584. Closing.
